### DISCUSSION TITLE
ArviZ showcase on Bokeh home page

### DISCUSSION THREAD

**Discussion by pavithraes** [arviz-devs/arviz#1883] [General]
Hi! I'm a part of the Bokeh core team. We'd love to feature ArviZ in the user showcase section on [bokeh.org](https://bokeh.org/), and we want to make sure the ArviZ community is willing and fine with this. :)

**Comment by OriolAbril** [comment]
That sounds amazing! Is there anything we have to do?

**Reply by pavithraes** [reply]
Yay, thanks!

Well, it'll be great if you can provide/point us to a bokeh plot you like (from the [example gallery](https://arviz-devs.github.io/arviz/examples/index.html) perhaps?), and share 2-4 sentences about ArviZ that we can add to the showcase.

We're also more than happy to do this ourselves, i.e., find/create an image and write the copy, if that's fine with you and the team!

**Reply by OriolAbril** [reply]
Perhaps one of the "plot_pair" examples?

* https://arviz-devs.github.io/arviz/examples/plot_pair_hex.html
* https://arviz-devs.github.io/arviz/examples/plot_pair_kde.html
* https://arviz-devs.github.io/arviz/examples/plot_pair_kde_hdi.html
* https://arviz-devs.github.io/arviz/examples/plot_pair_point_estimate.html

Anything that catches your eye from the example gallery should work perfectly too, don't feel tied to this.

As for summary sentences something along the lines of:

> ArviZ is a community-led package for exploratory analysis of Bayesian models in Python.
>
> It Includes functions for posterior analysis, data storage, sample diagnostics, model checking, and comparison. The goal is to provide backend-agnostic tools for diagnostics and visualizations of Bayesian inference.

**Reply by pavithraes** [reply]
I love it, thank you so much!

**Reply by pavithraes** [reply]
It's been added: https://bokeh.org/

Thanks again!
